Extensions/TextInput/textinputruntimeobject.ts

```diff
@@ -500,6 +536,36 @@ namespace gdjs {
     isFocused(): boolean {
       return this._renderer.isFocused();
     }
+    isSubmitted(): boolean {
+      return this._isSubmitted;
+    }
+
+    getMaxLength(): integer {
+      return this._maxLength;
+    }
+    setMaxLength(value: integer) {
+      this._maxLength = value;
+      this._renderer.updateMaxLength();
+    }
+    getPadding(): integer {
+      return this._padding;
+    }
+    setPadding(value: integer) {
+      this._padding = value;
+      this._renderer.updatePadding();
+    }
+
+    getTextAlign(): SupportedTextAlign {
+      return this._textAlign;
+    }
+
+    setTextAlign(newTextAlign: string) {
+      const lowercasedNewTextAlign = newTextAlign.toLowerCase();
+      if (lowercasedNewTextAlign === this._textAlign) return;
+
+      this._textAlign = parseTextAlign(lowercasedNewTextAlign);
```

Comment:
You can do the parsing as the first step and then you the returned value for the early return. Otherwise, you equality check above is wrong